mitosheet/mitosheet/api/get_imported_files_and_dataframes.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Saga Inc.
+# Distributed under the terms of the GPL License.
+from copy import copy
+import json
+import os
+from typing import Any, Dict, List, Optional
+
+import pandas as pd
+from mitosheet.types import StepsManagerType
+from mitosheet.step_performers.import_steps.simple_import import SimpleImportStepPerformer
+from mitosheet.step_performers.import_steps.excel_import import ExcelImportStepPerformer
+from mitosheet.step_performers.import_steps.dataframe_import import DataframeImportStepPerformer
+from mitosheet.step_performers.import_steps import is_import_step_type
+
+def get_index_from_possible_null_list(l: Optional[List[Any]], index: int) -> Optional[Any]:
```

Comment:
TODO: make this function more useful, cuz as of now it's always wrapped by the same call